### PR TITLE
Fixes #75 on entry for each file being compared

### LINKED ISSUE #75: Generate one entry in report for each file being compared

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs [hunk 1]
```diff
@@ -30,10 +30,7 @@ public override void Build(BatchComparisonResult comparisonResult, OSPSuiteTrack
             validationResultFor(comparisonResult)
          };
 
-         var fileComparisonResults = comparisonResult.FileComparisonResults.Where(x => x.State != ValidationState.Valid).ToList();
-         if (fileComparisonResults.Any())
-            objectsToReport.Add(new Paragraph(Assets.Reporting.FailedValidations));
-
+         var fileComparisonResults = comparisonResult.FileComparisonResults.OrderByDescending(x => x.State);
```
Comment (msevestre) [inline src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs]: So that Invalid will be displayed first in the list

Comment (Yuri05) [inline src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs]: That's good.
Can you also make sure that comparison results with the same `ValidationState `are sorted alphabetically by the `FileName `within their group?

File: src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs [hunk 2]
```diff
@@ -30,9 +30,9 @@ public override void Build(BatchComparisonResult comparisonResult, OSPSuiteTrack
             validationResultFor(comparisonResult)
          };
 
-         var fileComparisonResults = comparisonResult.FileComparisonResults.Where(x => x.State != ValidationState.Valid).ToList();
-         if (fileComparisonResults.Any())
-            objectsToReport.Add(new Paragraph(Assets.Reporting.FailedValidations));
+         var fileComparisonResults = comparisonResult.FileComparisonResults
+            .OrderByDescending(x => x.State)
+            .ThenBy(x => x.FileName);
```
Comment (msevestre) [inline src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs]: done

File: src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs (hunk 1 above)
Comment (msevestre) [inline src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs]: done

**Comment by msevestre** [comment]
Merging as I need to work on the same file.

**Pull request merged**